File: Models/Enums/OrderStatus.cs
using Swashbuckle.AspNetCore.Annotations;

[SwaggerSchema("Enumeration representing the status of an order")]
public enum OrderStatus
{
    Pending,    // Order has been created but not yet processed
    Processing, // Order is currently being processed
    Shipped,    // Order has been shipped to the customer
    Delivered,  // Order has been delivered to the customer
    Cancelled   // Order has been cancelled
}

File: models/enums/order_status.py
from enum import Enum

class OrderStatus(str, Enum):
    PENDING = "Pending"        # Order has been created but not yet processed
    PROCESSING = "Processing"  # Order is currently being processed
    IN_TRANSIT = "InTransit"   # Order has been shipped to the customer
    COMPLETED = "Completed"    # Order has been delivered to the customer
    CANCELLED = "Cancelled"    # Order has been cancelled

    def __str__(self) -> str:
        return self.value
